docs/src/main/tut/pull_request.md

```diff
@@ -11,6 +11,8 @@ with Github4s, you can:
 - [List pull requests](#list-pull-requests)
 - [List the files in a pull request](#list-the-files-in-a-pull-request)
 - [Create a pull request](#create-a-pull-request)
+- [List reviews](#list-reviews)
```

Comment:
This link won't work since you changed your heading to "List pull request reviews"